web/js/map-OpenLayers.js

```diff
@@ -791,6 +794,10 @@ $.extend(fixmystreet.utils, {
 
     $(function(){
 
+        if (!fixmystreet.map_type) {
```

Comment:
This test fails on Bexley and Lincolnshire and breaks the map for some reason.

Reply:
Okay, turns out map_type isn't always defined, I thought it was; I've switched to checking for the presence of the map div.